### DISCUSSION TITLE
在第三方软件中点击链接跳转

### DISCUSSION THREAD

**Discussion by HerstalR** [windingwind/zotero-better-notes#450] [Q&A] [answered]
在zotero中点击通过注释添加笔记之后，点击注释下的链接可以到指定的当前页面。而将markdown导出到第三方软件obsidian中再点击链接，却默认显示到zotero的文献库中的条目。有什么办法可以将点击链接指向指定的当前页面呢？

**Comment by windingwind** [comment, marked as answer]
批注导出后的格式是：内容(引用信息)
点击内容转跳pdf对应位置的批注，点击引用信息转跳库中位置。

**Reply by HerstalR** [reply]
感谢指导！
